statcode/statcode.py

```diff
@@ -280,12 +281,42 @@ def print_help():
     print(''.join([UNDERLINE, "Usage:", END, " $ statcode ", YELLOW, "status_code", END]))
 
 
+def print_all():
```

Comment:
Is there any way you can work this into the `App` object? I think it would help with keeping the UI code organized and easily readable. Maybe add a parameter to the object's `__init__` method for displaying all status codes.